pkg/apimodel/api_audit.go

```diff
@@ -9,17 +11,21 @@ type QueryAPIAudit struct {
 	HTTPPath       *string `query:"http_path"`
 	HTTPMethod     *string `query:"http_method"`
 	SentHTTPStatus *int    `query:"sent_http_status"`
+
+	Limit     *int    `query:"limit"`
+	Offset    *int    `query:"offset"`
+	OrderBy   *string `query:"order_by"`
+	OrderDesc bool    `query:"order_desc"`
 }
 
 // APIAudit is the response apimodel
 type APIAudit struct {
-	CreatedAt        time.Time `json:"created_at,omitempty"`
-	UpdatedAt        time.Time `json:"updated_at,omitempty"`
-	DeletedAt        time.Time `json:"deleted_at,omitempty"`
-	Plane            string    `json:"plane,omitempty"`
-	HashedGrantToken string    `json:"-"`
-	Domain           string    `json:"domain,omitempty"`
-	HTTPPath         string    `json:"http_path,omitempty"`
-	HTTPMethod       string    `json:"http_method,omitempty"`
-	SentHTTPStatus   int       `json:"sent_http_status,omitempty"`
+	CreatedAt        iso8601ms.Time `json:"created_at,omitempty"`
```

Comment:
We can discuss it and be adding this later, but we should evaluate adding IP and user agent as audit data

Reply:
will add in the next PR